receiver/dockerstatsreceiver/config.go

```diff
@@ -40,6 +42,8 @@ type Config struct {
 	// for the mapped name.  E.g. `io.kubernetes.container.name: container_spec_name`
 	// would result in a MetricDescriptor label called `container_spec_name` whose
 	// Metric DataPoints have the value of the `io.kubernetes.container.name` container label.
+	// Deprecated: [v0.61.0]
```

Comment:
These config options are still available for configuration, but you've removed the code that uses it. I can see this leading to non-intuitive behaviour for users.

Reply:
My mistake, I see now that you are translating the old config into the new 👍